docs/blogs/links-syphon/syphon.md

```diff
@@ -0,0 +1,151 @@
+# Mixing Metadata, Air and Water: Use the lakeFS Airflow Provider to Link Airflow Execution to lakeFS Data
+
+By [Ariel Shaqed (Scolnicov)]
+
+## Introduction
+
+"How do I integrate X with lakeFS" is an ever-green question on [lakeFS
+Slack][lakefs-slack].  lakeFS takes a "tooling-first" strategy to data
+management: it slots into your existing lineup of tools.  So a significant
+part of our work on lakeFS is devoted to leveraging lakeFS and these other
+tools to improve these integrations.  Our latest addition is Project Syphon!
+
+<img alt="Soda syphon, by Avi Nahmias, available from https://commons.wikimedia.org/wiki/User:Avin" src="https://upload.wikimedia.org/wikipedia/commons/1/1c/Soda_syphon_IMG_1505C.jpg" width=400/>
+
+Airflow was one of our earliest integrations: Itai published the original
+[Air and Water][airflow-air-and-water-blog-1] 2 years ago.  It's time to
```

Comment:
I'd be inclined to include a brief tl;dr of what Airflow is and why integrating it with lakeFS is a thing.